display/ssd1306.h

```diff
@@ -330,47 +345,48 @@ class StandardDisplayController : public DisplayControllerBase<Width, col_t>, Sa
       default:
       case SCREEN_UNSET:
       case SCREEN_OFF:
-	Clear();
+        Clear();
         return 3600000; // Long time!
 
       case SCREEN_STARTUP:
-	Clear();
+        Clear();
         // DrawText("==SabeR===", 0,15, Starjedi10pt7bGlyphs);
         // DrawText("++Teensy++",-4,31, Starjedi10pt7bGlyphs);
-	if (WIDTH < 128) {
-	  display_->DrawText("p-os", 0,15, Starjedi10pt7bGlyphs);
-	} else {
-	  display_->DrawText("proffieos", 0,15, Starjedi10pt7bGlyphs);
-	}
-	display_->DrawText(version,0,31, Starjedi10pt7bGlyphs);
-	if (HEIGHT > 32) {
-	  display_->DrawText("installed: ",0,47, Starjedi10pt7bGlyphs);
-	  display_->DrawText(install_time,0,63, Starjedi10pt7bGlyphs);
-	}
+        if (WIDTH < 128) {
+          display_->DrawText("p-os", 0,15, Starjedi10pt7bGlyphs);
+        } else {
+          display_->DrawText("proffieos", 0,15, Starjedi10pt7bGlyphs);
+        }
+        display_->DrawText(version,0,31, Starjedi10pt7bGlyphs);
+        if (HEIGHT > 32) {
+          display_->DrawText("installed: ",0,47, Starjedi10pt7bGlyphs);
+          display_->DrawText(install_time,0,63, Starjedi10pt7bGlyphs);
+        }
         next_screen_ = SCREEN_PLI;
-        return font_config.ProffieOSFontImageDuration;
+        return font_config.ProffieOSTextMessageDuration;
```

Comment:
backwards compatibility?
Something like:

if (font_config.ProffieOSTextMessageDuration > 0) return font_config.ProffieOSTextMessageDuration;
return font_config.ProffieOSFontImageDuration;

?

Reply:
My comment seems to keep disappearing.
This is addresses above.